vault/logical_system_quotas.go

```diff
@@ -34,9 +35,33 @@ func (b *SystemBackend) quotasPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
 					Callback: b.handleQuotasConfigUpdate(),
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
```

Comment:
seems to be no-content
https://github.com/hashicorp/vault/blob/4b8bd15e28ad3194159e7601fe76a994688988d4/vault/logical_system_quotas.go#L140
```suggestion
						http.StatusNoContent: {{
```